src/tersets.zig

```diff
@@ -128,13 +128,17 @@ pub fn isWithinErrorBound(
     error_bound: f32,
 ) bool {
     if (uncompressed_values.len != decompressed_values.len) {
+        std.debug.print("\nHere not the same size\n", .{});
         return false;
     }
 
     for (0..uncompressed_values.len) |index| {
         const uncompressed_value = uncompressed_values[index];
         const decompressed_value = decompressed_values[index];
-        if (@abs(uncompressed_value - decompressed_value) > error_bound) return false;
+        if (@abs(uncompressed_value - decompressed_value) > error_bound) {
+            std.debug.print("\n{} and {}\n", .{ uncompressed_value, decompressed_value });
```

Comment:
Maybe remove the leftover debugging code and make the `if` expression a one-liner again.

Reply:
Ohh no, I forgot about it.